registration-processor/pre-processor/registration-processor-packet-validator-stage/src/main/java/io/mosip/registration/processor/stages/packet/validator/PacketValidateProcessor.java

```diff
@@ -455,7 +457,41 @@ public MessageDTO process(MessageDTO object, String stageName) {
 
 	}
 
-		private boolean isValidSupervisorStatus(MessageDTO messageDTO) {
+	private void setPacketCreatedDateTime(InternalRegistrationStatusDto registrationStatusDto) throws ApisResourceAccessException, PacketManagerException, JsonProcessingException, IOException {
+		LogDescription description = null;
+		try {
+			description = new LogDescription();

```

Comment:
Do we really need LogDescription here ? It is used in the process method because it can consumed in the finally which is not the case here

Reply:
Description Removed.